Gems/ROS2/Code/Source/RobotImporter/Utils/RobotImporterUtils.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "AzCore/Component/ComponentBus.h"
+#include "AzCore/std/string/string.h"
+#include "RobotImporter/URDF/UrdfParser.h"
+#include <AzCore/IO/SystemFile.h>
+#include <AzCore/Math/Transform.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/function/function_template.h>
+
+namespace ROS2
+{
+    namespace
+    {
+        const AZStd::function<bool(const AZStd::string&)> fileExistsCall = [](const AZStd::string& filename) -> bool
```

Comment:
Why not just make this a regular static function?
```suggestion
        static bool FileExistsCall(const AZStd::string& filename)
```

Reply:
That is what I propose here:
``` 
 namespace
    {
        static inline bool FileExistsCall(const AZStd::string& filename)
        {
            return AZ::IO::SystemFile::Exists(filename.c_str());
        };
    }

Reply:
Done